extract/Dockerfile

```diff
@@ -61,7 +61,9 @@ RUN apt update && apt upgrade -y && \
 COPY --from=builder /extract-build/extract-*.war /usr/local/tomcat/webapps/extract.war
 COPY tomcat/conf/context.xml /usr/local/tomcat/conf/context.xml
 COPY script.py /extract/script.py
+COPY entrypoint.sh /extract/entrypoint.sh
 
 RUN mkdir -p /var/log/extract
 RUN chown -R www-data:www-data /usr/local/tomcat/ /var/log/extract/ /var/extract/orders /extract/
-USER 33
```

Comment:
We should keep the `USER` instruction to avoid having a container running as root

Reply:
Running it as www-data, restored it with a textual name